settings.py

```diff
@@ -496,6 +496,9 @@
     ]
 }
 
+# Note: if you change this, make sure to update the cronjob in commcarehq-ansible
+RESTORE_PAYLOAD_DIR = os.path.join(FILEPATH, 'restore_payloads')
```

Comment:
Agree we generally treat code_root as something we can blow away and replace with a fresh clone. The fact that it's .gitignore'd means that it'll survive the `git clean -ffd` we run on deploy, but still I agree it'd be nice to save somewhere else. We could go off whatever convention we set with touchforms sessions (though I guess those are being moved to a DB). Looking at `touchforms` role in ansible it's: `{{ www_home }}/data/touchforms`.

To do that you'd have to add something analogous in ansible for creating the `{{ www_home }}/data/restore_payloads` directory (like https://github.com/dimagi/commcarehq-ansible/blob/3bf708a6be2830f67abca9d2fbab549c00abef54/ansible/roles/touchforms/tasks/main.yml#L30 for touchforms)